src/lib/registration/connect_helpers.rb

```diff
@@ -121,11 +121,11 @@ def self.catch_registration_errors(message_prefix: "",
         false
       rescue ::Registration::ServiceError => e
         log.error("Service error: #{e.message % e.service}")
-        Yast::Report.Error(_(e.message) % e.service)
+        report_pkg_error(_(e.message) % e.service)
```

Comment:
Only for curiosity: Can this message be correctly translated? The string is generated at runtime, so how to know what is the message?

Reply:
The message is defined here: https://github.com/yast/yast-registration/blob/6dbf98a245e104e3135cf1d87d305476e020dcf1/src/lib/registration/sw_mgmt.rb#L321

The `N_()` method only marks the string for translation, it returns the original (untranslated string). It's later translated via the `_()` call. See http://yastgithubio.readthedocs.io/en/latest/localization/#translating-constants

The reason is that the `log.error` above will log the original untranslated message. That's for easier debugging, reading errors logged in a non-Latin based language is tricky.